examples/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM golang:1.19-alpine as builder
+
+RUN apk --no-cache add git
+
+RUN go install -trimpath go.k6.io/xk6/cmd/xk6@latest
+
+RUN  xk6 build --output "/tmp/k6" --with github.com/grafana/xk6-browser
+
+FROM alpine:3.14
+
+ENV XK6_HEADLESS=true
+
+RUN apk add chromium
```

Comment:
Maybe we should version pin the `chromium` package as well? Via an `ARG`, so that it could be changed at build time.

Reply:
done :white_check_mark: